main/include/OpenThreadConfig.h

```diff
@@ -41,6 +46,29 @@
 //
 #include "openthread-core-nrf52840-config.h"
 
-#endif // OPENTHREAD_PLATFORM_CONFIG_H
+// ==============================================================
+//          CONFIG TOBLE
+// ==============================================================
+#define OPENTHREAD_CONFIG_ENABLE_TOBLE 1
```

Comment:
This will only enable ToBLE when third_party/openthread is replaced with the ToBLE branch of OpenThread.  Master will ignore all ToBLE related configuration options.

I'm not fully understanding the statement "Given that it prevents connecting to a device over ToBLE after its paired".  This flag enables connecting a device over ToBLE after it is paired.  Did you mean WoBLE?

Reply:
Sorry, I did mean WoBLE.

I think this should be disabled by default, so that when ToBLE lands in OpenThread it won't turn on by default.

Reply:
Disabled by default.